src/store/mutations.js

```diff
@@ -21,18 +21,11 @@ const ReverseRecords = new ethers.Contract(REACT_APP_ENS_CONTRACT, ensABI, provi
 
 // TODO: Refactor to render as it returns data rather than waiting all in batch
 export async function getEnsData(ownerIds){
-  const chunked = _.chunk(ownerIds, 50)
+  const chunked = _.chunk(ownerIds, 1200)
```

Comment:
This value should probably be a constant

Reply:
+1000